lib/rubocop/cop/rspec/stubbed_mock.rb

```diff
@@ -81,7 +81,7 @@ def offending_argument_range(source_map)
           Parser::Source::Range.new(
             source_map.expression.source_buffer,
             source_map.dot.begin_pos,
-            source_map.end.end_pos
+            source_map.end&.end_pos || source_map.expression.end_pos
```

Comment:
This seems to work for all cases:
```suggestion
            source_map.expression.end_pos
```
…also for `#offending_block_range` below.

Reply:
It does, thanks!